Examples/MAX78000/CNN/actionrecognition-demo/camera_util.c

```diff
@@ -0,0 +1,385 @@
+ * Copyright (C) 2023 Maxim Integrated Products, Inc., All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL MAXIM INTEGRATED BE LIABLE FOR ANY CLAIM, DAMAGES
+ * OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * Except as contained in this notice, the name of Maxim Integrated
+ * Products, Inc. shall not be used except as stated in the Maxim Integrated
+ * Products, Inc. Branding Policy.
+ *
+ * The mere transfer of this software does not imply any licenses
+ * of trade secrets, proprietary technology, copyrights, patents,
+ * trademarks, maskwork rights, or any other form of intellectual
+ * property whatsoever. Maxim Integrated Products, Inc. retains all
+ * ownership rights.
+ *
+ ******************************************************************************/
+
+/***** Includes *****/
+#include <stdio.h>
+#include <stdint.h>
+#include "mxc.h"
+#include "mxc_device.h"
+#include "mxc_delay.h"
+#include "uart.h"
+#include "led.h"
+#include "board.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include "cnn.h"
+#include "lp.h"
+#include "camera.h"
+#include "dma.h"
+#include "dma_regs.h"
+#include "camera_util.h"
+#include "tft_config.h"
+#include "rtc.h"
+
+#ifdef BOARD_FTHR_REVA
+#include "tft_ili9341.h"
+#endif
+
+int slaveAddress;
+int id;
+int dma_channel;
+
+static uint32_t *data_addr0 = (uint32_t *)0x50400700;
+//static uint32_t * data_addr1 = (uint32_t *) 0x50408700;
+//static uint32_t * data_addr2 = (uint32_t *) 0x50410700;
+static uint32_t *data_addr3 = (uint32_t *)0x50418700;
+
+//static uint32_t * data_addr4 = (uint32_t *) 0x50800700;
+//static uint32_t * data_addr5 = (uint32_t *) 0x50808700;
+static uint32_t *data_addr6 = (uint32_t *)0x50810700;
+//static uint32_t * data_addr7 = (uint32_t *) 0x50818700;
+
+//static uint32_t * data_addr8 = (uint32_t *) 0x50c00700;
+static uint32_t *data_addr9 = (uint32_t *)0x50c08700;
+//static uint32_t * data_addr10 = (uint32_t *) 0x50c10700;
+//static uint32_t * data_addr11 = (uint32_t *) 0x50c18700;
```

Comment:
Will the commented-out lines be used in the future?

Reply:
No, they were used in an earlier version. Deleting.